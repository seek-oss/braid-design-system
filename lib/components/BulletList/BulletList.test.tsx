import React from 'react';
import { BulletList } from '..';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('BulletList', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <BulletList>
            <li>Bullet</li>
            <li>Bullet</li>
            <li>Bullet</li>
          </BulletList>
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
