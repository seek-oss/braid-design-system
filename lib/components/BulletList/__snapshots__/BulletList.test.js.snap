// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulletList Theme: jobStreet Default 1`] = `
<ul
  class="Box__root borderColor__standard reset__ul"
>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
</ul>
`;

exports[`BulletList Theme: seekAnz Default 1`] = `
<ul
  class="Box__root borderColor__standard reset__ul"
>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
</ul>
`;

exports[`BulletList Theme: seekAsia Default 1`] = `
<ul
  class="Box__root borderColor__standard reset__ul"
>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
</ul>
`;

exports[`BulletList Theme: wireframe Default 1`] = `
<ul
  class="Box__root borderColor__standard reset__ul"
>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
  <li>
    Bullet
  </li>
</ul>
`;
